index.js

```diff
@@ -1 +1,71 @@
 // implement your API here
+const express = require('express');
+const server = express();
+const User = require('./data/db.js');
+
+server.use(express.json())
+
+// Read - Get the list of Users
+server.get('/api/users', (req, res) => {
+    User.find()
+        .then(data => {
+            console.log('happy path');
+            res.status(200).json(data);
+        })
+        .catch(({ code, message }) => {
+            res.status(500).json({ err: 'The users information could not be retrieved.' });
+        });
+});
+
+// Post - Creates a user using the information sent inside the request body.
+server.post('/api/users', (req, res) => {
+    const newUser = req.body;
+    console.log(newUser)
+    User.insert(newUser)
+    .then(addedUser => {
+        if(newUser.name === null || newUser.bio === null){
```

Comment:
Here we should validate the `newUser` before we `insert` it into the database. We should also compare it to `undefined` rather than `null`:
```js
if(newUser.name === undefined || newUser.bio === undefined){
  res.status(400).json({ err: 'Please provide name and bio for the user.' });
} else {
  // Insert user into database
}
```